src/java.base/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -108,6 +110,54 @@ public final class SunEntries {
 
         // common attribute map
         HashMap<String, String> attrs = new HashMap<>(3);
+        if (FIPSConfigurator.enableFips()) {
+            // FIPS supported
+            attrs.put("ImplementedIn", "Software");
```

Comment:
This should be included unconditionally.

Reply:
updated